docs/content/installation/uninstalling_hostdefiner.md

```diff
@@ -0,0 +1,14 @@
+# Uninstalling the host definer
```

Comment:
We should add a note that emphasize that upon uninstall no host/s will be removed from the storage even these hosts that have been configured on storage by the hostdefiner (that is being uninstalled).

Reply:
done

Reply:
@rivkap I do not see the note.

Reply:
Saw it thanks